phase1/vsphere/README.md

```diff
@@ -79,17 +193,52 @@ You have a Kubernetes cluster!
 **Notes**
 If you want to launch another cluster while keeping existing one then clone the kubernetes-anywhere and follow the steps above.
 
-#### Get Cluster Info:
+#### Next Steps:
 
 First set KUBECONFIG to access cluster using kubectl:
+
 ```shell
 export KUBECONFIG=phase1/vsphere/.tmp/kubeconfig.json
 ```
+
 You will get cluster information when you run:
 ```shell
 kubectl cluster-info
 ```
 
+To access the dashboard after successful instllation of kubernetes cluster. There are 2 options.
+
+* Run ```kubectl proxy``` outside the container spawned by ```make docker-dev```
+```
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.4.7/bin/linux/amd64/kubectl
```

Comment:
Remove kubectl installtion from here. Add it to pre-requisites and link to google page.